bigtable/google/cloud/bigtable/table.py

```diff
@@ -553,7 +557,7 @@ def _do_mutate_retryable_rows(self):
 
 def _create_row_request(table_name, row_key=None, start_key=None, end_key=None,
                         filter_=None, limit=None, end_inclusive=False,
-                        app_profile_id=None):
+                        app_profile_id=None, row_set=None):
```

Comment:
The `row_set` parameter is not documented in this method.

Reply:
Documented now.